apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SeedStep.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Flex } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+// import { StorylyInstanceID } from "@ledgerhq/types-live";
+
+// import { StorylyWrapper } from "~/renderer/components/Storyly";
+import { StepText } from "./shared";
+
+export type SeedPathStatus =
+  | "choice_new_or_restore"
+  | "new_seed"
+  | "choice_restore_direct_or_recover"
+  | "restore_seed"
+  | "recover_seed";
+
+export type Props = {
+  seedPathStatus: SeedPathStatus;
+};
+
+const SeedStep = ({ seedPathStatus }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <Flex flexDirection="column">
+      <StepText mb={6}>
+        {seedPathStatus === "new_seed"
+          ? t("syncOnboarding.manual.seedContent.newSeed")
+          : seedPathStatus === "choice_restore_direct_or_recover"
+          ? t("syncOnboarding.manual.seedContent.choiceRestoreDirectOrRecover")
+          : seedPathStatus === "restore_seed"
+          ? t("syncOnboarding.manual.seedContent.restoreSeed")
+          : seedPathStatus === "recover_seed"
+          ? t("syncOnboarding.manual.seedContent.recoverSeed")
+          : t("syncOnboarding.manual.seedContent.choiceNewOrRestore")}
+      </StepText>
+      {/* <StorylyWrapper instanceID={StorylyInstanceID.recoverySeed} /> */}
```

Comment:
<img width="229" alt="Screenshot 2023-03-16 at 10 09 26" src="https://user-images.githubusercontent.com/91890529/225568749-f2839ef8-4d43-45e9-bcdb-1714beed1189.png">


this thing is supposed to be displayed in the `"new_seed"` state I think

Reply:
ah i'm stupid it's LLD ok